app/scripts/components/common/page-header-legacy/index.tsx

```diff
@@ -0,0 +1,360 @@
+  gap: ${variableGlsp()};
+  padding: ${variableGlsp(1, 0)};
+
+  ${media.largeUp`
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 0;
+  `}
+`;
+
+const NavBlock = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  gap: ${glsp(0.25)};
+
+  ${media.largeUp`
+    flex-direction: row;
+    align-items: center;
+    gap: ${glsp(1.5)};
+  `}
+`;
+
+const SROnly = styled.a`
+  height: 1px;
+  left: -10000px;
+  overflow: hidden;
+  position: absolute;
+  top: auto;
+  width: 1px;
+  color: ${themeVal('color.link')};
+  &:focus {
+    top: 0;
+    left: 0;
+    background-color: ${themeVal('color.surface')};
+    padding: ${glsp(0.25)};
+    height: auto;
+    width: auto;
+  }
+`;
+
+const SectionsNavBlock = styled(NavBlock)`
+  /* styled-component */
+`;
+
+const GlobalNavBlockTitle = styled(Overline).attrs({
+  as: 'span'
+})`
+  ${visuallyHidden}
+  display: block;
+  padding: ${variableGlsp(1, 1, 0.25, 1)};
+  color: currentColor;
+  opacity: 0.64;
+
+  ${media.largeUp`
+    padding: 0;
+  `}
+`;
+
+const GlobalMenu = styled.ul`
+  ${listReset()}
+  display: flex;
+  flex-flow: column nowrap;
+  gap: ${glsp(0.5)};
+
+  ${media.largeUp`
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    gap: ${glsp(1.5)};
+  `}
+`;
+
+interface PageHeaderProps {
+  mainNavItems: NavItem[];
+  subNavItems: NavItem[];
+  logo: ReactElement;
+  linkProperties: LinkProperties;
+}
+
+function PageHeader(props: PageHeaderProps) {
+  const { mainNavItems, subNavItems, logo, linkProperties } = props;
+  const { isMediumDown } = useMediaQuery();
+  const [globalNavRevealed, setGlobalNavRevealed] = useState(false);
+
+  const globalNavBodyRef = useRef<HTMLDivElement>(null);
+  // Click listener for the whole global nav body so we can close it when clicking
+  // the overlay on medium down media query.
+  const onGlobalNavClick = useCallback((e) => {
+    if (!globalNavBodyRef.current?.contains(e.target)) {
+      setGlobalNavRevealed(false);
+    }
+  }, []);
+
+  useEffect(() => {
+    // Close global nav when media query changes.
+    // NOTE: isMediumDown is returning document.body's width, not the whole window width
+    // which conflicts with how mediaquery decides the width.
+    // JSX element susing isMediumDown is also protected with css logic because of this.
+    // ex. Look at GlobalNavActions
+    if (!isMediumDown) setGlobalNavRevealed(false);
+  }, [isMediumDown]);
+
+  const closeNavOnClick = useCallback(() => {
+    setGlobalNavRevealed(false);
+  }, []);
+
+  function skipNav(e) {
+    // a tag won't appear for keyboard focus without href
+    // so we are preventing the default behaviour of a link here
+    e.preventDefault();
+    // Then find a next focusable element in pagebody,focus it.
+    const pageBody = document.getElementById(PAGE_BODY_ID);
+    if (pageBody) {
+      pageBody.focus();
+    }
+  }
+
+  return (
+    <>
+      <SROnly href='#' onClick={skipNav}>
```

Comment:
Thanks for putting this back 🙇 But I am seeing a problem that this button takes me back to the main page no matter where I was before. - (this is mostly the same thing of what we had before and I wonder why this doesn't work the way before?)